src/account/SignIn.jsx

```diff
@@ -0,0 +1,69 @@
+
+import React from 'react';
+import { getAuth, signInWithEmailAndPassword } from 'firebase/auth';
+
+
+const auth = getAuth();
```

Comment:
You may need to do something like this here:

```
import { initializeApp } from "firebase/app";
const firebaseConfig = {
  apiKey: "your-api-key",
  authDomain: "your-auth-domain",
  projectId: "your-project-id",
  storageBucket: "your-storage-bucket",
  messagingSenderId: "your-messaging-sender-id",
  appId: "your-app-id"
};
const app = initializeApp(firebaseConfig);
const auth = getAuth(app);
```

Reply:
Figure out the issue. It was an issue with importing and moving certain files